scripts/pages/view.py

```diff
@@ -223,17 +223,17 @@ def displayGeneral(hide=False):
     primarytitle, primarytitleDetail, primaryformat = view_sections[0]
     printSectionHeader(primarytitle, primarytitleDetail, hide);
 
-    print "<form id='tiqitBugEdit' action='edit.py' method='post' onSubmit='return prepareForm();'>"
-    print cls.getFormat(primaryformat) % args
+    print "<form id='tiqitBugEdit' action='edit.py' method='post' onSubmit=''>";
```

Comment:
Semicolon not needed in Python code
```suggestion
    print "<form id='tiqitBugEdit' action='edit.py' method='post' onSubmit=''>"
```